datafusion/src/logical_plan/plan.rs

```diff
@@ -212,6 +212,15 @@ pub enum LogicalPlan {
         /// The logical plan
         input: Arc<LogicalPlan>,
     },
+    /// Drops a table.
+    DropTable {
+        /// The table name
+        name: String,
+        /// If the table exists
+        if_exist: bool,
+        /// Dummy schema
+        schema: DFSchemaRef,
```

Comment:
i wonder if it's valuable to actually just attach the fetched table schema

Reply:
otherwise you can just remove this placeholder

Reply:
Why do we need a schema for dropping the table?

Reply:
yea, it sounds more an empty schema is better here.